<?xml version='1.0'?>
<launch>
	<node pkg="mavel" type="tf_odom_node" name="tf_odom" clear_params="true" output="screen">
		<param name="frame_world" value="world" type="str"/>
		<param name="frame_mav" value="fcu" type="str"/>

		<param name="topic_odom" value="/odom" type="str"/>

		<param name="lookup_rate" value="50.0" type="double"/>
		<param name="velocity_interval" value="0.25" type="double"/>
		<param name="velocity_filtering" value="0.4" type="double"/>
	</node>
</launch>
